### DISCUSSION TITLE
Using flit in PDM

### DISCUSSION THREAD

**Discussion by thmsklngr** [pdm-project/pdm#2268] [General]
Hi there,

I try to build my first (internal) installable module and want to use flit as build backend. Configuration is done right with PDM, but what I wonder is, that even though I never installed flit on my system (neither in the virtual environment I'm currently working in nor in the site-packages of the Python version I use PDM with), `pdm build` works as expected and makes a package (source and wheel) of my 2 lines of code I've written so far. :)

So, what's the deal? Why do I specify a build backend and it seems that it's not used? Or is it somehow built in into PDM?

Please enlighten me. :)

Regards, Thomas

**Comment by pawamoy** [comment]
(quick answer on mobile)

The build frontend creates a temporary venv in which it installs the build requirements, and then runs the build backend in this venv.

You should get hints of what is happening if you run pdm build in verbose mode.
